Jenkinsfile

```diff
@@ -6,29 +6,19 @@
 
 @Library(['private-pipeline-library', 'jenkins-shared']) _
 
-def workDir = "./"
 
 dockerizedBuildPipeline(
-  buildImageId: "${sonatypeDockerRegistryId()}/cdi/golang-1.14:1",
+  pathToDockerfile: 'jenkins/Dockerfile',
   prepare: {
     githubStatusUpdate('pending')
-    runSafely '''
-    curl -O https://releases.hashicorp.com/terraform/0.13.3/terraform_0.13.3_linux_amd64.zip
```

Comment:
The prepare step does not run in the docker container so this isn't doing you much good.